test_scripts/API/DeleteCommand/ATF_DeleteCommand_UI_DeleteCommand_WARNINGS.lua

```diff
@@ -0,0 +1,190 @@
+-- in this test case when UI.DeleteCommand gets WARNINGS is checked
+-- 1. Used preconditions: App is activated and registered SUCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends DeleteCommand
+-- SDL -> HMI: resends VR.DeleteCommand with all valid params, UI.DeleteCommand with imageType of cmdIcon which is not supported by HMI
+-- HMI -> SDL: VR.DeleteCommand (SUCCESS), UI.DeleteCommand (WARNINGS)
+--
+-- Expected result:
+-- SDL -> MOB: appID: (WARNINGS, success: true: DeleteCommand)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.."/"
+local ServerAddress = commonFunctions.read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,_)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+        :Times(1)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+function Test:Precondition_AddSubMenu()
+  --mobile side: sending AddSubMenu request
+  local cor_id_add_submenu = self.mobileSession:SendRPC("AddSubMenu",
+  {
+    menuID = 1000,
+    position = 0,
+    menuName ="Commandpositive"
+  })
+  --hmi side: expect UI.AddSubMenu request
```

Comment:
@vvvakulenko, please remove such comments, as it's understand from other command.
Please apply this to all comments for "hmi side" / "mobile side".